docs-ref-conceptual/TOC.yml

```diff
@@ -117,6 +117,9 @@
     - name: Configure settings
       href: azure-cli-Configuration.md
       displayName: configure, defaults, environment, variables
+    - name: Work with persisted parameters
+      href: param-persist-howto.md
+      displayName: az config persist-param, az config
```

Comment:
`az config param-persist` is deprecated, see https://github.com/Azure/azure-cli/issues/24726

Reply:
This PR has an old version of the TOC.  This TOC entry was removed in PR https://github.com/MicrosoftDocs/azure-docs-cli/pull/3933